retrofit/src/test/java/retrofit2/CallTest.java

```diff
@@ -781,24 +781,24 @@
     Object a = new Object() {
       @Override public String toString() {
         writeCount.incrementAndGet();
-        throw new OutOfMemoryError("Broken!");
+        throw new AssertionError("Broken!");
```

Comment:
nit: `AssertionError` doesn't sound like "recoverable" :D

Reply:
yeah, maybe rename the tests to "nonFatal" and "fatal" instead of "recoverable" and "unrecoverable"?

Reply:
yep, I definitely like that it matches the `throwIfFatal` naming!